portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,29 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> messages;
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Enoch!");
+    messages = new ArrayList<String>();
+    messages.add("It\'s a beautiful day in the neighborhood!");
+    messages.add("Let\'s make the most of this beautiful day!");
+    messages.add("I\'ve always wanted to have a neighbor just like you!");
+    String json = convertToJson(messages);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  private String convertToJson(List<String> messages) {
```

Comment:
1. Please add comment to function. It should be clear what does it return.
2. Another question: Have you decided on json return structure or it was part of requirements?
If it was your decision then please explain why.
3. Use string formatting instead of concatenation.

Reply:
Json has ability to store arrays. Will it be better to use this json feature?

Reply:
Ok, I just added a comment to the function. I decided on using a json object, because I wasn't sure how to access json array elements. I just wanted to push a working version of the code before I ended the day on Friday, but I can test out using array elements to see if it works.

Reply:
Try to use arrays.